CHANGELOG.rst

```diff
@@ -2,6 +2,353 @@
+          type in the ZGW APIs registration plugin.
+        * [:backend:`4344`] You can now select which Objects API group to use rather than "the first one"
+          being used always.
+    * StUF-ZDS:
+
+        * [:backend:`4319`] You can now provide a custom document's title for StUF-ZDS via the component's
+          configuration.
+        * [:backend:`4762`] The cosigner identifier (BSN) is now included in the created case.
+* Prefill plugins:
+
+    * Added documentation for product prefill in user manual (examples are also provided).
+
+    * Objects API:
+
+        *
+
+* Payment plugins:
+
+    * Ogone:
+
+        * [:backend:`3457`] Custom `title` and `com` parameters can now be defined in Ogone payment plugin.
+* [:backend:`4785`] Updated the eHerkenning metadata generation to match the latest standard version(s).
+* [:backend:`4930`] It's now possible to export registered submission metadata via the form statistics
+  admin page. This can be based on specific date range.
+* The documentation of Open Forms is now available for offline access too. You can find a PDF link
+  on the bottom of the page.
+* [:backend:`2173`] The map component now supports using a different background/tile layer
+* [:backend:`4321`] Forms can now have a submission limit. The SDK displays appropriate messages when
+  this limit is reached.
+* [:backend:`4907`] Improved developer installation documentation.
+* [:backend:`4895`] Ensured that custom email headers are used for confirmation emails and cosign
+  request.
+* [:backend:`4859`] Improved form state handling.
+* [:backend:`4789`, :backend:`4788`] Added `django-setup-configuration` to configure Open Forms to
+  Open Zaak and Objecten.
+* [:backend:`4787`] It's now possible to load a confguration file via the ``setup_configuration``
+  management command. Additional information/instructions are provided in
+  :ref:`installation_configuration_cli`.
+* Updated documentation concerning frontend toolchains and formio search strategies.
+* [:backend:`4798`] Made the confirmation box consistent with other modals and improved the UX.
+* [:backend:`4320`] Improved the cosign flow and the texts used in cosign flows, while adding more
+  flexibility:
+
+    - You can now use templates specifically for cosigning for the confirmation screen content,
+      with the ability to include a 'cosign now' button.
+    - You can now use templates specifically for cosigning for the confirmation email subject and content.
+    - When links are used in the cosign request email, the cosigner can now directly click through without
+      having to enter a code to retrieve the submission.
+    - Updated the default templates with better text/instructions.
+    - Updated translations of improved texts.
+* [:backend:`4815`] Changed submission removal limit to 0, allowing submissions to be deleted after 0 days
+  (i.e. on the same day).
```

Comment:
```suggestion
* [:backend:`4815`] The minimum submission removal limit is now 0 days, allowing submissions to be deleted on the
   same day they were created.
```

Reply:
This was copied over from the alpha version. Should I change it?

https://github.com/open-formulieren/open-forms/blob/master/CHANGELOG.rst?plain=1#L47

Reply:
Yeah :)